### ISSUE TITLE
NoSuchMethodError from FregeParser

### ISSUE THREAD

**Issue by carymrobbins** [carymrobbins/intellij-frege#3]
Running the test suite (or just `FregeParserTest`) causes a `NoSuchMethodError`.

Frege version: 3.24.80

```
java.lang.NoSuchMethodError: frege.prelude.PreludeBase$TST.$gt$gt$eq(Lfrege/run8/Func$U;Lfrege/run8/Func$U;)Lfrege/run8/Func$U;

  at com.haskforce.frege.lang.parser.FregeParser.psiParse(FregeParser.java:139)
  at com.haskforce.frege.lang.parser.FregePsiParser.parse(FregePsiParser.java:16)
  at com.intellij.psi.tree.ILazyParseableElementType.doParseContents(ILazyParseableElementType.java:85)
  at com.intellij.psi.tree.IFileElementType.parseContents(IFileElementType.java:43)
  at com.intellij.psi.impl.source.tree.LazyParseableElement.a(LazyParseableElement.java:172)
  at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:212)
  at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:36)
  at com.intellij.psi.impl.source.tree.SharedImplUtil.acceptChildren(SharedImplUtil.java:189)
  at com.intellij.psi.impl.source.PsiFileImpl.acceptChildren(PsiFileImpl.java:820)
  at com.intellij.testFramework.ParsingTestCase$6.visitElement(ParsingTestCase.java:331)
  at com.intellij.psi.PsiElementVisitor.visitFile(PsiElementVisitor.java:34)
  at com.intellij.extapi.psi.PsiFileBase.accept(PsiFileBase.java:70)
  at com.intellij.testFramework.ParsingTestCase.ensureParsed(ParsingTestCase.java:328)
  at com.intellij.testFramework.ParsingTestCase.doTest(ParsingTestCase.java:227)
  at com.haskforce.frege.lang.parser.FregeParserTestBase.doTest(FregeParserTestBase.java:14)
  at com.haskforce.frege.lang.parser.FregeParserTest.testExample00002(FregeParserTest.java:6)
  at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
```

**Comment by carymrobbins** [comment]
Note that this doesn't seem to happen when simply _running_ the plugin.

**Comment by carymrobbins** [comment]
I've created a script, [tools/run-tests](https://github.com/carymrobbins/intellij-frege/blob/e03b17f95b702ddfd68d69160521d74456e83cf5/tools/run-tests) for getting around this issue. It runs the tests directly without going through IntelliJ's JUnit run configuration. Possibly this is an issue with IntelliJ.

I've logged an issue with JetBrains regarding this - https://youtrack.jetbrains.com/issue/IDEA-158393
